### PR TITLE
“Ajit_Assignment_22_(20+10 mins)”

### PR DESCRIPTION
**Pull request by AjitSJune20**
image
image

### CODE REVIEW

File: src/ajitSJune20/TestPalindromeNumber.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package ajitSJune20;
+
+public class TestPalindromeNumber {
+
+	static boolean isPalindromeInteger(int number) {
```
Comment (KartikeyDJune20) [inline src/ajitSJune20/TestPalindromeNumber.java:5]: in my opinion .. u are checking reverse number with provided number two times .. we can set both the things reverse+palindrome in main also..

Comment (AjitSJune20) [reply src/ajitSJune20/TestPalindromeNumber.java:5]: yes I could have written every thing in main but for standard practice and guideline of Kirishna, we prefer to create method to perform operation/action. 
For re-usability of my code , I have written two separate method to check reverse and palindrome.  we can use same program to find reverse or palindrome. In our earlier assignment we wrote a program to find reverse, I enhanced same program to find palindrome.

Comment (KartikeyDJune20) [reply src/ajitSJune20/TestPalindromeNumber.java:5]: Okay np.. bro.. its good u used the same function for both..

Comment (KrishnaKTechnocredits) [reply src/ajitSJune20/TestPalindromeNumber.java:5]: Code Reusability looks good to me @AjitSJune20

File: src/ajitSJune20/TestPalindromeString.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package ajitSJune20;
+
+public class TestPalindromeString {
+	static String revStr = "";
+
+	static boolean isPalindromeString(String string) {
+		if (string.equalsIgnoreCase(reverse(string))) {
```
Comment (KrishnaKTechnocredits) [inline src/ajitSJune20/TestPalindromeString.java:7]: better to use equals method.

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code looks good to me.

**Pull request merged**